src/client/app/components/about.ts

```diff
@@ -50,10 +64,31 @@ export class About implements AfterViewInit {
     currentChosenLanguage = '';
     originalLanguage = '';
 
+    configs = [];
+    selectedConfig = '';
+
+    constructor(
+        private _effects: UIEffects,
+        private _changeDetector: ChangeDetectorRef
+    ) {}
+
     ngAfterViewInit() {
         this.availableLanguages = getAvailableLanguages();
         this.currentChosenLanguage = getDisplayLanguage();
         this.originalLanguage = this.currentChosenLanguage;
+        this.configs.push(
+            { name: 'production', value: 'Production' },
+            { name: 'insiders', value: 'Beta' },
+            { name: 'edge', value: 'Alpha' },
+        );
+
+        // user can only navigate to localhost if they've sideloaded local manifest
+        if (environment.current.config.name === config['local'].name ||
```

Comment:
Nit: can you isolate the large parenthetical inside the `if` statement into its own variable.  I.e., 

    showLocalConfig = ...;
    if (showLocalConfig) { ... }